packages/kbn-securitysolution-io-ts-list-types/src/request/import_exception_list_schema/index.test.ts

```diff
@@ -0,0 +1,129 @@
+    const decoded = importExceptionsListSchema.decode(payload);
+    const checked = exactCheck(payload, decoded);
+    const message = pipe(checked, foldLeftRight);
+
+    expect(getPaths(left(message.errors))).toEqual([]);
+    expect(message.schema).toEqual(getImportExceptionsListSchemaDecodedMock());
+  });
+
+  test('it should NOT accept an undefined for "list_id"', () => {
+    const payload = getImportExceptionsListSchemaMock();
+    // @ts-expect-error
+    delete payload.list_id;
+    const decoded = importExceptionsListSchema.decode(payload);
+    const checked = exactCheck(payload, decoded);
+    const message = pipe(checked, foldLeftRight);
+    expect(getPaths(left(message.errors))).toEqual([
+      'Invalid value "undefined" supplied to "list_id"',
+    ]);
+    expect(message.schema).toEqual({});
+  });
+
+  test('it should NOT accept an undefined for "description"', () => {
+    const payload = getImportExceptionsListSchemaMock();
+    // @ts-expect-error
+    delete payload.description;
+    const decoded = importExceptionsListSchema.decode(payload);
+    const checked = exactCheck(payload, decoded);
+    const message = pipe(checked, foldLeftRight);
+    expect(getPaths(left(message.errors))).toEqual([
+      'Invalid value "undefined" supplied to "description"',
+    ]);
+    expect(message.schema).toEqual({});
+  });
+
+  test('it should NOT accept an undefined for "name"', () => {
+    const payload = getImportExceptionsListSchemaMock();
+    // @ts-expect-error
+    delete payload.name;
+    const decoded = importExceptionsListSchema.decode(payload);
+    const checked = exactCheck(payload, decoded);
+    const message = pipe(checked, foldLeftRight);
+    expect(getPaths(left(message.errors))).toEqual([
+      'Invalid value "undefined" supplied to "name"',
+    ]);
+    expect(message.schema).toEqual({});
+  });
+
+  test('it should NOT accept an undefined for "type"', () => {
+    const payload = getImportExceptionsListSchemaMock();
+    // @ts-expect-error
+    delete payload.type;
+    const decoded = importExceptionsListSchema.decode(payload);
+    const checked = exactCheck(payload, decoded);
+    const message = pipe(checked, foldLeftRight);
+    expect(getPaths(left(message.errors))).toEqual([
+      'Invalid value "undefined" supplied to "type"',
+    ]);
+    expect(message.schema).toEqual({});
+  });
+
+  test('it should NOT accept value of "true" for "immutable"', () => {
+    const payload = { ...getImportExceptionsListSchemaMock(), immutable: true };
+
+    const decoded = importExceptionsListSchema.decode(payload);
+    const checked = exactCheck(payload, decoded);
+    const message = pipe(checked, foldLeftRight);
+    expect(getPaths(left(message.errors))).toEqual([
+      'Invalid value "true" supplied to "immutable"',
+    ]);
+    expect(message.schema).toEqual({});
+  });
+
+  test('it should accept any partial fields', () => {
+    const payload = {
+      ...getImportExceptionsListSchemaMock(),
```

Comment:
When you use a splat like this, you should type the payload like so:

```ts
    const payload: ImportExceptionListItemSchema = {
      ...getImportExceptionsListItemSchemaMock(),
      id: '123',
      namespace_type: 'single',
      comments: [],
      os_types: [],
      tags: ['123'],
      created_at: '2018-08-24T17:49:30.145142000',
      created_by: 'elastic',
      updated_at: '2018-08-24T17:49:30.145142000',
      updated_by: 'elastic',
      tie_breaker_id: '123',
      _version: '3',
      meta: undefined,
    };
```

The splats turn off the typing altogether which is why you should add them when you do a splat.